readme.md

```diff
@@ -1,14 +1,22 @@
-# [Argus Eyes](http://arguseyes.io/) [![npm version](https://badge.fury.io/js/argus-eyes.svg)](https://www.npmjs.org/package/argus-eyes)
+# DEPRECATED - [Argus Eyes](http://arguseyes.io/) [![npm version](https://badge.fury.io/js/argus-eyes.svg)](https://www.npmjs.org/package/argus-eyes)
 
-| Linux & Mac OS X | Windows |
-| --- | --- |
+| No Maintenance Intended | Linux & Mac OS X | Windows |
+| --- | --- | --- |
+| [![No Maintenance Intended](http://unmaintained.tech/badge.svg)](http://unmaintained.tech/) |
```

Comment:
I would not add the badge inside the table, better to put it after the title, next to the `npm verson` badge.